server/views/match/placementRequests/occupancyView/view.njk

```diff
@@ -15,6 +16,16 @@
 
 {% block content %}
     <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
+    {% set detailsHTML %}
+        {{ govukSummaryList({
+            rows: MatchUtils.occupancyViewSummaryListForMatchingDetails(totalCapacity, dates, placementRequest, essentialCharacteristics)
```

Comment:
This pattern of creating a utility that's used directly in the template has been used everywhere historically, but I think we should move away from it: it relies on the integration tests to be covered properly, and these are somewhat slow, and it doesn't help discoverability as one has to do a manual search for that utility method name to figure out if it is used in a template, versus having the IDE let you discover usages. 

A more testable approach would be to call those utilities in the controller directly, and simply return directly usable data/variables to the template. 

Not necessarily one to change right now, but let's have a chat about this when we can!

Reply:
have refacored as suggested, agree this is much nicer! Keep the views as dumb as possible